indeex.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="stylesheet.css">
+    <header>
+        <!-- section which is not needed -->
+    <div class="logo"><!-- logo will be put here --></div>
+    <nav> 
+        <ul>
+            <li><!--Home --></li>
+            <li><!--About --></li>
+            <li><!--Services --></li>
+            <li><!-- Gallery --></li>
+            <li><!--Blogs --></li>
+            <li><!-- Contact--></li>
+
+        </ul>
+    </nav> 
+    <h3><a href="#">Login</a></h3>
+    </header>
+</head>
+<body>
+<!-- container practice -->
+
+<div class="main-container">
+
+    <div class="first-container">
+            <div class="item1">
+                <h2>About </h2>
+                    <p>its Mishal firdous here, i am a software engineer, Previsiosly having one year CMS expertices in wordpress,
+                         wix and shopify.Which plays a critical role in the technology industry. Through these content management 
+                         system design would more eye cautching although responsible for designing, creating, and maintaining software
+                          that meets the needs of businesses and users in various domains, including finance, healthcare, e-commerce, 
+                          entertainment, and many others. Software engineers work with programming languages, algorithms, and software 
+                          development methodologies to build applications and systems that drive modern society.</p>
+            </div>
+            <div class="item2">
+                <h2>Experince</h2>
+            </div>
+    </div>
+
+    <div class="sec-container">
+       
+        <div class="item3">
+            <img src="c:\Users\MishalFirdous\Downloads\WhatsApp Image 2023-10-20 at 3.42.56 AM.jpeg">
+            <h2>Mishal Firdous</h2>
+            <ul class="a">
+               <li><label for="file">Wordpress, Wix, Shopify </label>
+<progress id="file" value="90" max="100"> 90% </progress></li>
+
+<li><label for="file">HTML5 ,CSS3, Bootstrap</label>
+    <progress id="file" value="70" max="100"> 32% </progress></li>
+
+    <li><label for="file">Adobe Photo Shop </label>
+        <progress id="file" value="30" max="100"> 32% </progress></li>
+
+            </ul>
+        </div>
+    </div>
+</div>
+
+</body>
+<footer class="footer">
+    <h3><a href="https://www.linkedin.com/in/mishal-firdous-b99a0a131">Footer</a></h3>
```

Comment:
Name this as LinkedIn instead of footer.

Reply:
resolved!

Reply:
I cant see this change!

Reply:
i changed the class name with LinkedIn.